crates/pipeline_manager/src/db/mod.rs

```diff
@@ -2066,20 +2087,51 @@ impl Storage for ProjectDB {
                     .await?;
         Ok(TenantId(res.get(0)))
     }
+
+    async fn create_compiled_binary_ref(
```

Comment:
Should there also be an API to delete refs to garbage collected binaries?

Reply:
Yes, I'll add that.